spec/models/vote_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+
+RSpec.describe Vote, type: :model do
+  context "associations" do
+    it { should belong_to(:place) }
```

Comment:
Do you have any thoughts on using expectations instead of `should`s? Re: https://github.com/rspec/rspec-expectations/blob/main/Should.md

Reply:
I'm only using them here, because that's how we've used [shoulda matchers](https://github.com/thoughtbot/shoulda-matchers) in the past